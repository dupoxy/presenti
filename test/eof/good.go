/* #nb:
A Good presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/
*/

// A eof good Example
package main

import (
	"fmt"
)

/* #nb: * Bla bla */

func main() {
	fmt.Println("bad")
}

/* #nb: * Bla bla */
