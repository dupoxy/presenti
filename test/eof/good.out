# DO NOT EDIT GENERATED by presenti
A Good presenti Example

The presenti Authors
https://www.github.com/dupoxy/presenti/

* Bla bla
* Bla bla
